components/userDetail/essentialInfo.tsx

```
@@ -0,0 +1,173 @@
+
+        <View className="bg-[#F1F2F4] h-[1px] my-[12px]" />
+
+        <View className="flex flex-row items-center gap-x-[12px] flex-1">
+          <Text className="Medium14 text-colorFont">잠버릇</Text>
+          <Text className="Medium14 leading-6 text-basicFont flex-1">
+            {data.memberStatDetail.sleepingHabits &&
+              data.memberStatDetail.sleepingHabits.join(', ')}
+          </Text>
+        </View>
+
+        <View className="bg-[#F1F2F4] h-[1px] my-[12px]" />
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)